src/utils/helpers.ts

```diff
@@ -57,3 +57,42 @@ export function getCoords({
 
   return new Matrix2D().translate(tx, ty).scale(zoom).inverse();
 }
+
+/**
+ * Given a nodeId to find, a list of nodes to check against, and an optional parentId of the node
+ * find the node from the list of nodes
+ */
+export function findNestedNode(
+  nodeId: string,
+  children: any[],
```

Comment:
Missing type

Reply:
There is no declared type for this. `children` in this case is the `ElkLayout.children` which has a type of `any[]`